exe/technologist

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby
```

Comment:
I've been using `exe` because that's automatically used by the gemspec as binary:

~~~ruby
spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
~~~

And at least bundler uses that too.